backend/coins/btc/handlers/handlers.go

```diff
@@ -222,65 +221,12 @@ func (handlers *Handlers) postExportTransactions(_ *http.Request) (interface{},
 		}
```

Comment:
Wouldn't you want to return this error from the handler? If file.Close fails, the data might not have been written to the file. I'd move this out of defer, to the very end after ExportCSV.

Reply:
Done.

Should keep in mind though that errors returned like this from handlers are not handled by the frontend in the Qt build. It's a long-standing design issue that needs to be resolved some day.

Currently, when a frontend must handle errors, we return an error object like `{ "success": false, "code" ... }`.